### ISSUE TITLE
recurrent training code

### ISSUE THREAD

**Issue by Henry831** [facebookresearch/pifuhd#63]
i have a question about HGPIFuNetwNML.py code，why the def get_error has a parameter named gamma，the have bug when i run the code，because the loss only need pred and label that two input，and the loss function is defined to MSE ，i see the paper about the loss function is BCE

**Comment by shunsukesaito** [comment]
The default parameters used in the HGPIFuNetwNML are not the ones used for training. 
Please refer to here.
https://github.com/facebookresearch/pifuhd/blob/85ed42cecf4eea78f092168fedd1d6913a8ab52d/lib/net_util.py#L86

**Comment by Henry831** [comment]
why you say the gamma is not a ones used for training?
the training is not the two part?After training coarse and to train fine part?

**Comment by Henry831** [comment]
thanks for your response
and i have another question
how can i get the points_nml and labels_nml and if i need to product front and back normal map by others github or the network can product by itself

**Comment by shunsukesaito** [comment]
The normals are computed inside the code base.

**Comment by zener90818** [comment]
@Henry831 Hi, Did you have the training code  about the PifuHD,  Could you please share with me, many thanks.
